tests/cypress/tests/create_policy_field_validation.spec.js

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2020 Red Hat, Inc. */
+/// <reference types="cypress" />
+
+describe('RHACM4K-2349 - GRC UI: [P1][Sev1][policy-grc] Create policy page: Check policy field validations', () => {
+
+  const ID = Cypress.env('RESOURCE_ID')
+  const errorMsgRegExp = /Invalid name due to Kubernetes naming restriction.The name must meet the following requirements:• contain no more than 253 characters• contain only lowercase alphanumeric characters, '-' or '.'• start with an alphanumeric character• end with an alphanumeric character/
+//  const errorMsgRegExp = /Invalid name due to Kubernetes naming restriction.The name must meet the following requirements:/
+  const alertMsgRegExp = /Name already exists: pressing 'Create' will prompt to update the existing policy/
+  const namePatterns = [
+                         'this-is-n,ot-a-valid-name',
+                         '-this-is-not-a-valid-name',
+                         'this-is-not-a-valid-name-',
+                         'this-is-a-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-long-name-that-is-too-long-and-should-not-work-when-its-put-into-the-name-field'
+                       ]
+  // while we could use the next definition, due to a bug bz#1936431 better use a unique policy name
+  //const longValidName = 'this-is-a---really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-long-valid-name-that-should-work-when-its-put-into-the-name-field'`
+  const longValidName = `this-is-a---really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-long-valid-name-that-should-work-when-its-put-into-the-name-field-${ID}`
```

Comment:
@kkaarreell Please consider this change as well https://github.com/open-cluster-management/grc-ui/pull/494
and I would recommend to use loop or function to generate this string, won't recommend  to keep such long string in spec file.

Reply:
Yes, It depends what gets it first. ;-) I guess we should prioritize @dhaiducek 's fix, so I will make this PR WIP.